### PR TITLE
Update SciPy-bundle-2020.02 with numpy-1.18.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/698c0a5cdb0e42182fc8280a2f602560 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/39dfc93905f12a4e3f43d99c1c444ec0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/eb179838b9656f609ca5bdf59be98c89 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2491.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9c51af0c23c7c4c1932cc7ae41d2a03c for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/1266a86110b0a9a8aaa84195258eaf55 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/cc7c773564dc9b1a01a557f7f32d80d1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

File: easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2020.02-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -21,9 +21,9 @@ exts_default_options = {'source_urls': [PYPI_SOURCE]}
 
 # order is important!
 exts_list = [
-    ('numpy', '1.18.1', {
+    ('numpy', '1.18.2', {
```
Comment (boegel) [inline easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2020.02-foss-2020a-Python-3.8.2.eb]: @akesandgren This warrants also changing the `SciPy-bundle` version to `2020.03` (same goes for #10112)

Comment (akesandgren) [reply easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2020.02-foss-2020a-Python-3.8.2.eb]: Done in #10118

**Pull request merged**